sagemaker/lambda_function.py

```diff
@@ -0,0 +1,34 @@
+LAMBDA_FUNCION_CODE = """
```

Comment:
Why is this in string format?

Reply:
This code for lambda goes as a string into the CF template right now, so I thought it would be better as a string and we also have to substitute the endpoint name here too.